examples/animation-translates/index.html

```diff
@@ -8,13 +8,13 @@
   </head>
   <body>
     <a-scene>
-
-      <a-camera position="0 0 8" cursor-visible="false"></a-camera>
+      <a-entity position="0 0 8">
```

Comment:
[http://localhost:9000/examples/animation-translates/](http://localhost:9000/examples/animation-translates/)

this was already busted on `dev` in both Chrome and Firefox:

![screenshot 2016-02-17 03 25 38](https://cloud.githubusercontent.com/assets/203725/13108156/499751f4-d526-11e5-85bb-038a7b33d2f2.png)
<img width="1363" alt="screenshot 2016-02-17 03 25 44" src="https://cloud.githubusercontent.com/assets/203725/13108157/4997a474-d526-11e5-99f5-e349c5a2de0f.png">

and in this branch the Chrome one still is broken, but the Firefox one looks good (nice work):

![screenshot 2016-02-17 03 13 00](https://cloud.githubusercontent.com/assets/203725/13108143/38683a2e-d526-11e5-8869-6a271c50ec13.png)
<img width="1363" alt="screenshot 2016-02-17 03 13 04" src="https://cloud.githubusercontent.com/assets/203725/13108144/386b1eba-d526-11e5-9e42-e69a86b0df0f.png">